bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -650,6 +650,9 @@ private function addAssetNode(ArrayNodeDefinition $rootNode): void
                 ->arrayNode('metadata')
                 ->addDefaultsIfNotSet()
                     ->children()
+                        ->scalarNode('alt_prefix')
+                            ->defaultValue('')
+                        ->end()
```

Comment:
Same or some of the text from the docs into a `->info('what this does')` here would make a lot of sense.

Reply:
nice idea, thank you!